week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,54 @@
+// ############### Part One #################
+// ========= Elements to be manipulated =========
+let jumptron  = document.querySelector(".jumbotron");
+let blueBtn = document.querySelector("#blueBtn");
+let orangeBtn = document.querySelector("#orangeBtn");
+let greenBtn = document.querySelector("#greenBtn");
+let dntBtn = document.getElementsByClassName("btn-lrg")[0];
+let vlnBtn = document.getElementsByClassName("btn-lrg")[1];
```

Comment:
This is fine. General advice, if we can, use more precise selectors to elements you are looking for e.g. here ".jumbotron .btn-primary". That would make your code more resilient/flexible against modification in the HTML page. Changing of the class of an element is less likely than changes of in their text, but the order can change too and in that case, your code would be inverted.

So all I mean it better selector you have, less likely to change the code in case of HTML changes. But it doesn't mean that you have to write something super long very precise selector 😄  that could be also super fragile.